comm/src/cli.rs

```diff
@@ -0,0 +1,36 @@
+use clap::{crate_authors, crate_description, crate_name, crate_version, App, Arg};
+
+pub(crate) fn create_app<'help>() -> App<'help> {
+    App::new(crate_name!())
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about(crate_description!())
+        .mut_arg("help", |help| help.help("Display help information.").short('?'))
+        .mut_arg("version", |v| v.help("Display version information."))
+        .arg(
+            Arg::new("1")
+                .short('1')
+                .help("Suppress lines unique to file1.")
+                
+        )
+        .arg(
+            Arg::new("2")
+                .short('2')
+                .help("Suppress lines unique to file2.")
+        )
+        .arg(
+            Arg::new("3")
+                .short('3')
+                .help("Suppress lines common to both files.")
+        )
```

Comment:
So far we have been adding short and long on all flags. Since those don't have a long version on any other implementation, you are free to create one